frontend/src/components/UserSearch.vue

```diff
@@ -269,9 +269,6 @@ import UsersRepository from "@/api/UsersRepository";
 import ClientsRepository from "@/api/ClientsRepository";
 import app_config from '@/loadconfig';
 
-const options = {dateStyle: 'short'};
-const formatDate = new Intl.DateTimeFormat(undefined, options).format;
```

Comment:
I think the only change needed was to define the locale like, `const formatDate = new Intl.DateTimeFormat('en-CA', options).format;`.